bus/routes.go

```diff
@@ -166,9 +166,7 @@ func (b *Bus) consensusStateHandler(jc jape.Context) {
 }
 
 func (b *Bus) consensusNetworkHandler(jc jape.Context) {
-	jc.Encode(api.ConsensusNetwork{
-		Name: b.cm.TipState().Network.Name,
-	})
+	jc.Encode(*b.cm.TipState().Network)
```

Comment:
Sneaky stuff. This is fine but might require a UI update. cc @alexfreska

Reply:
@ChrisSchinnerl actually the UI does not use this endpoint / its not in our SDKs, so no breaking change but I will add the endpoint to our SDKs.